pkg/process/model.go

```diff
@@ -14,40 +15,44 @@ type Department struct {
 	Schedules []DepartmentSchedule // time schedule for a whole department
 }
 
+// Registration
 // registration table
 type Registration struct {
-	ID         uint `gorm:"primaryKey"`
-	Doctor     account.Doctor
-	Patient    account.Patient
-	Department Department
-	Date       time.Time
-	HalfDay    HalfDayEnum            // TODO: a validator for registration, only half day is allowed
-	Status     RegistrationStatusEnum `gorm:"default:'committed'"`
+	ID           uint `gorm:"primaryKey"`
+	DoctorID     uint
```

Comment:
Hmm so you choose not to use foreign key? That's ok.

Reply:
It should be the foreign key.

I'll fix it.

Reply:
fixed